web/src/beta/features/Editor/tabs/map/LeftPanel/Layers/LayerItem.tsx

```diff
@@ -90,10 +91,19 @@ const LayerItem = ({
           onBlur={handleEditExit}
         />
       ) : (
-        <Text size="body">{layerTitle}</Text>
+        <TitleContainer>
+          <Text size="body">{layerTitle}</Text>
+        </TitleContainer>
       )}
     </ListItem>
   );
 };
 
+const TitleContainer = styled.div`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 200px;
```

Comment:
Since we can resize the left/right panels, we shouldnt set a max-width. It should grow to show more of the hidden layer name as the panel grows.
Also, the `...` isn't showing. The name just gets hard cutoff

Reply:
Fixed.